cumulus/polkadot-parachain/polkadot-parachain-lib/src/common/chain_spec.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Chain spec primitives.
+
+use frame_support::{Deserialize, Serialize};
+use sc_chain_spec::{ChainSpec, ChainSpecExtension};
+use std::fmt::Debug;
+
+/// Helper trait used for loading/building a chain spec starting from the chain ID.
+pub trait LoadSpec: Debug {
```

Comment:
why `Debug`?

Reply:
The `Cli` struct required this, and I thought it was related to clap. But seems to work without it. Removed it.

Reply:
Did you in the end not decide to remove it?